lib/paginator.js

```diff
@@ -102,8 +103,35 @@ Paginator.prototype.prepateQuery = function() {
 
   this.whereComp = function(qb) {
     _.forIn(fixedWhere, function(value, key) {
-      debugWhere('adding where (', key, _this.comps[key], value, ')');
-      qb.andWhere(key, _this.comps[key], value);
+
```

Comment:
json queries are too specific to postgres , maybe a it's better idea allow include raw conditions